docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/vertex.md

```diff
@@ -13,12 +13,68 @@ redirect_from:
+
+The Spryker Vertex app offers the following features that are worth considering when comparing it to the default Spryker [Tax Management capability](/docs/pbc/all/tax-management/{{page.version}}/tax-management.html) or the [Avalara integration](/docs/pbc/all/tax-management/{{page.versoin}}/base-shop/third-party-integrations/avalara/avalara.html):
+
+- *Compliance in states or countries with complex tax calculations*: Takes into account various tax jurisdictions and rates applicable to different states and countries. This helps a lot with tax calculations if you sell in states or countries where tax calculations are complex. 
+- *Dynamic tax calculation*: As tax rules or rates change in Vertex, these changes are applied to customer’s quote during the checkout process.
+- *Reduction of manual errors*: Helps to automate tax calculations hence reducing manual errors and saving time.
+- *Tax exemption management*: You can manage tax exemptions and certificates efficiently, ensuring that exempt customers are not charged taxes during the checkout process, while maintaining proper documentation.
+- *Tax reporting and filing*: You have the option of sending invoice requests to Vertex after an order is paid. You can generate accurate tax reports which helps in the filing of sales tax returns across multiple jurisdictions, reducing the risk of audit and penalties.
+- *Application of custom tax rules to products*: You can implement custom tax rules to accommodate unique product categorizations or specific tax regulations that apply to your business. The Vertex Integration provides a means for taxes to be calculated using these rules.
+
+## Using the Vertex app in Spryker-based projects
+
+To start using the Spryker Vertex app in your project, you need to do the following:
+
+1. Make sure your project is ACP-enabled. See [App Composition Platform installation](/docs/acp/user/app-composition-platform-installation.html) for details.
+2. Make sure you use NET prices in your project, as Vertex app only works with NET prices. GROSS prices aren't supported.
+3. Get an account with [Vertex](https://www.vertexinc.com/). Contact the Spryker Support team or your Customer Success Manager if you need support getting a Vertex account.
+4. Install the Vertex app. See [Install Vertex](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/install-vertex.html) for details.
+5. Configure the Vertex app. See [Configure Vertex](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/configure-vertex.html) for details.
+
+Once you have installed and configured the Vertex app, the taxes will be automatically calculated based on the address from where the order is shipped. In case of a Marketplace, Vertex uses the merchant warehouse address to calculate taxes.
```

Comment:
Rephrase this line to: Once you have installed and configured the Vertex app, the taxes will be calculated by Vertex during the checkout process. See the video below to understand the experience for the end user

---> Insert Video showing end user journey

Data Sent to Vertex for Tax Calculation
OOTB, we send the following data for Tax Calculation:

1. Shipping address
2. Sku
3. Warehouse address ( also includes Merchant Warehouse address for Marketplace model)

Note: Vertex uses an hierarchy structure for tax determination. The highest in the structure is the Company Code. It is important to setup your company code in Vertex and add it on your project as it determines what rate applies